### ISSUE TITLE
Does swa support model with groupnorm?

### ISSUE THREAD

**Issue by npmhung** [pytorch/contrib#35]
I'm trying to use swa, but when looking at the code, I only saw the batchnorm update.
How about groupnorm? Does swa affect groupnorm in any way?
